frontend/src/pages/StudentProfile.jsx

```
@@ -0,0 +1,407 @@
+    userType: ""
+  });
+  const {user} = useAuth();
+  const userId = user?.userId;
+  console.log(profileData)
+  const handleInputChange = (e) => {
+    const { name, value } = e.target;
+    
+    if (name === 'salaryExpectation') {
+      setProfileData({
+        ...profileData,
+        [name]: parseFloat(value) || 0.0, // Ensure it's a valid float
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)